lib/wallet.js

```diff
@@ -123,6 +137,70 @@ class Wallet extends ethers.Wallet {
 
         return [approvalTxReceipt, depositTxReceipt];
     }
+
+
+    /**
+      * Returns the wallet instance address
+      */
+    getAddress() {
+        return ethers.Wallet.prototype.getAddress.bind(this)();
+    }
+
+    /**
+      * Returns the wallet instance on-chain ETH balance
+      * @param {string} [blockTag] - A block number to calculate from
+      */
+    getBalance(blockTag) {
+        return ethers.Wallet.prototype.getBalance.bind(this)(blockTag);
+    }
+
+    /**
+      * Returns the wallet instance on-chain transaction count
+      * @param {string} [blockTag] - A block number to calculate from
+      */
+    getTransactionCount(blockTag) {
+        return ethers.Wallet.prototype.getTransactionCount.bind(this)(blockTag);
+    }
+
+    /**
+      * Signs and broadcasts an Ethereum transaction to the network
+      * @param {object} transaction - An unsigned Ethereum transaction
+      */
+    sendTransaction(transaction) {
+        return ethers.Wallet.prototype.sendTransaction.bind(this)(transaction);
+    }
+
+    /**
+     * @private - invoke bound to instance.
+     * Creates signing methods and derives an address from a private key and assigns them to this Wallet instance
+     * @param {string} privateKey - An Ethereum hex encoded private key
+     */
+    _initializeFromPrivateKey(privateKey) {
```

Comment:
This is just "private by convention", please make it a proper private function. There is no need for anyone external to have access to this function.